components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/handler/request/impl/DefaultRequestCoordinator.java

```diff
@@ -545,6 +561,20 @@ protected void unwrapResponse(CommonAuthResponseWrapper responseWrapper, String
         }
     }
 
+    private boolean isPromptRequest(HttpServletRequest request) {
+
+        return StringUtils.isNotBlank(request.getParameter(PROMPT_ID_PARAM))
+                && TRUE.equals(request.getParameter(PROMPT_RESP_PARAM));
```

Comment:
shall we boolean parse this

Reply:
Fixed in https://github.com/wso2/carbon-identity-framework/pull/6647/commits/ffb0217e864a5bf0248aea542ae14d3e57475659